cellarium/ml/models/scvi.py

```diff
@@ -0,0 +1,521 @@
+                * ``"poisson"``: :class:`~scvi.distributions.Poisson`.
+        latent_distribution:
+            Distribution to use for the latent space. One of the following:
+                * ``"normal"``: isotropic normal.
+                * ``"ln"``: logistic normal with normal params N(0, 1).
+        encode_covariates:
+            If ``True``, covariates are concatenated to gene expression prior to passing through
+            the encoder(s). Else, only gene expression is used.
+        deeply_inject_covariates:
+            If ``True`` and ``n_layers > 1``, covariates are concatenated to the outputs of hidden
+            layers in the encoder(s) (if ``encoder_covariates`` is ``True``) and the decoder prior to
+            passing through the next layer.
+        batch_representation:
+            ``EXPERIMENTAL`` Method for encoding batch information. One of the following:
+                * ``"one-hot"``: represent batches with one-hot encodings.
+                * ``"embedding"``: represent batches with continuously-valued embeddings using
+                :class:`~scvi.nn.Embedding`.
+            Note that batch representations are only passed into the encoder(s) if
+            ``encode_covariates`` is ``True``.
+        use_batch_norm:
+            Specifies where to use :class:`~torch.nn.BatchNorm1d` in the model. One of the following:
+                * ``"none"``: don't use batch norm in either encoder(s) or decoder.
+                * ``"encoder"``: use batch norm only in the encoder(s).
+                * ``"decoder"``: use batch norm only in the decoder.
+                * ``"both"``: use batch norm in both encoder(s) and decoder.
+            Note: if ``use_layer_norm`` is also specified, both will be applied (first
+            :class:`~torch.nn.BatchNorm1d`, then :class:`~torch.nn.LayerNorm`).
+        use_layer_norm:
+            Specifies where to use :class:`~torch.nn.LayerNorm` in the model. One of the following:
+                * ``"none"``: don't use layer norm in either encoder(s) or decoder.
+                * ``"encoder"``: use layer norm only in the encoder(s).
+                * ``"decoder"``: use layer norm only in the decoder.
+                * ``"both"``: use layer norm in both encoder(s) and decoder.
+            Note: if ``use_batch_norm`` is also specified, both will be applied (first
+            :class:`~torch.nn.BatchNorm1d`, then :class:`~torch.nn.LayerNorm`).
+        use_size_factor_key:
+            If ``True``, use the :attr:`~anndata.AnnData.obs` column as defined by the
+            ``size_factor_key`` parameter in the model's ``setup_anndata`` method as the scaling
+            factor in the mean of the conditional distribution. Takes priority over
+            ``use_observed_lib_size``.
+        use_observed_lib_size:
+            If ``True``, use the observed library size for RNA as the scaling factor in the mean of the
+            conditional distribution.
+        library_log_means:
```

Comment:
These can be computed via a separate one-pass model, `OnePassLibrarySizeStatistics`